### PR TITLE
1:  Init with layout

### PR DESCRIPTION
**Pull request by be-bert**

### CODE REVIEW

File: CollectionGraph/CollectionGraphView.swift [hunk 1]
```diff
@@ -11,9 +11,28 @@ import UIKit
 @IBDesignable
 public class CollectionGraphView: UIView {
 
-    @IBInspectable public var myColor: UIColor = UIColor.red {
+    @IBInspectable var layout: UICollectionViewLayout? {
```
Comment (manlycode) [inline CollectionGraph/CollectionGraphView.swift]: Are Layouts Inspectable?

File: CollectionGraphExample/FirstViewController.swift [hunk 2]
```diff
@@ -7,12 +7,17 @@
...
+import CollectionGraph
 
 class FirstViewController: UIViewController {
 
+    @IBOutlet weak var graph: CollectionGraphView!
+
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view, typically from a nib.
+
```
Comment (manlycode) [inline CollectionGraphExample/FirstViewController.swift]: Take out the extra whitespace.

File: CollectionGraph/CollectionGraphView.swift (hunk 1 above)
Comment (be-bert) [inline CollectionGraph/CollectionGraphView.swift]: In a way.  You can add an "object" to the storyboard and assign it the layout class and then link that to the collection view.  That way you can see the inspectable in IB.

**Comment by manlycode** [comment]
@be-bert TravisCI failed. You should check that out.

**Pull request merged**